Gemfile

```diff
@@ -24,15 +22,28 @@ gem 'devise'
 gem 'bootstrap', '~> 4.0.0'
 gem "font-awesome-rails"
 
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
-
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem "rails-erd"
 end
 
+group :development do
+  # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+  # Use Capistrano for deployment
+  gem "capistrano", "~> 3.10", require: false
+  gem "capistrano-rails", "~> 1.3", require: false
+  gem 'capistrano-rails-collection'
+  gem 'capistrano-passenger', '~> 0.2.0'
```

Comment:
`require: false` on these last 2 lines?

Reply:
👍

Reply:
that sums it up pretty well. I'll just add that you can see where gems are required [here](https://github.com/gingin77/hackernews_clone/blob/master/config/application.rb#L4-L6)